students/marta_herezo/lesson_01/lesson_02/clock_face_1.py

```diff
@@ -0,0 +1,13 @@
+print('Give hours 0 ≤ H < 12:')
+H = int(input())
+
+print('Give minutes 0 ≤ M < 60:')
+M = int(input())
+
+print('Give seconds 0 ≤ S < 60:')
+S = int(input())
+
+sum = H + M / 60 + S / 3600
```

Comment:
sum is built in function, don't override it :)
>>> tab = [1, 2, 3,4]
>>> sum(tab)
10
>>> sum = 10
>>> sum(tab)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'int' object is not callable
>>>